src/Configuration/MetaData/MetaDataManager.php

```diff
@@ -12,7 +12,7 @@ class MetaDataManager extends Manager
      */
     public function getDefaultDriver()
     {
-        return 'annotations';
+        return 'xml';
```

Comment:
Isnt the attributes driver a more natural default driver? Symfony uses this as default.

Reply:
The most technically correct answer is `xml`.  However most users of Doctrine use the easy way out and use attributes, now annotations.  If the goal of this project is to provide the most simple possible Doctrine metadata configuration then `annotations` is the right answer.  Given the history of this project recommending annotations, I agree attributes would be the simple, easy, and best default.